bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -219,41 +220,51 @@ def main():
+    parser.add_argument('-args', '--arguments', help='Additional arguments to pass into wtperf')
+    parser.add_argument('-ops', '--operations', help='List of operations to report metrics for')
     parser.add_argument('-v', '--verbose', action="store_true", help='be verbose')
     args = parser.parse_args()
 
     if args.verbose:
         print('WTPerfPy')
         print('========')
         print("Configuration:")
-        print("  WtPerf path:              {}".format(args.wtperf))
-        print("  Environment:              {}".format(args.env))
-        print("  Test path:                {}".format(args.test))
-        print("  Home base:                {}".format(args.home))
-        print("  Addition arguments(file): {}".format(args.arg_file))
-        print("  Git root:                 {}".format(args.git_root))
-        print("  Outfile:                  {}".format(args.outfile))
-        print("  Runmax:                   {}".format(args.runmax))
-        print("  JSON info                 {}".format(args.json_info))
-        print("  Reuse results:            {}".format(args.reuse))
-        print("  Brief output:             {}".format(args.brief_output))
+        print("  WtPerf path:       {}".format(args.wtperf))
+        print("  Environment:       {}".format(args.env))
+        print("  Test path:         {}".format(args.test))
+        print("  Home base:         {}".format(args.home))
+        print("  Batch file:        {}".format(args.batch_file))
+        print("  Arguments:         {}".format(args.arguments))
+        print("  Operations:        {}".format(args.operations))
+        print("  Git root:          {}".format(args.git_root))
+        print("  Outfile:           {}".format(args.outfile))
+        print("  Runmax:            {}".format(args.runmax))
+        print("  JSON info          {}".format(args.json_info))
+        print("  Reuse results:     {}".format(args.reuse))
+        print("  Brief output:      {}".format(args.brief_output))
 
     if args.wtperf is None:
         sys.exit('The path to the wtperf executable is required')
     if args.test is None:
         sys.exit('The path to the test file is required')
     if args.home is None:
         sys.exit('The path to the "home" directory is required')
-    if args.arg_file and not os.path.isfile(args.arg_file):
-        sys.exit("arg_file: {} not found!".format(args.arg_file))
+    if args.batch_file and not os.path.isfile(args.batch_file):
+        sys.exit("batch_file: {} not found!".format(args.batch_file))
+    if args.batch_file and (args.arguments or args.operations):
+        sys.exit("A batch file (-bf) should not be defined at the same time as -ops or -args")
 
     json_info = json.loads(args.json_info) if args.json_info else {}
+    arguments = json.loads(args.arguments) if args.arguments else []
+    operations = json.loads(args.operations) if args.operations else []
```

Comment:
How about this?
```
    arguments = json.loads(args.arguments) if args.arguments else None
    operations = json.loads(args.operations) if args.operations else None
```
Since we check the `arguments` and `operations` against `None` not `[]`. [here](https://github.com/wiredtiger/wiredtiger/blob/2c42dba3a0abbe1691e1549c5d4f4cb9e3a0f423/bench/wtperf/wtperf_run_py/wtperf_run.py#L89)

Reply:
👍 Good catch! Changed to None in be211